src/core/reactor.ts

```diff
@@ -1202,7 +1202,8 @@ export abstract class Reactor extends Component {
         }
       } else {
         // IN to OUT
-        return false;
+        // FIXME: Make this direct feedthrough false after the generated code deletes it.
```

Comment:
I don't quite understand this comment.

Reply:
I made some changes and left the description of them in this comment https://github.com/lf-lang/reactor-ts/pull/186#discussion_r1275695631.